ansible/roles/network/files/hooks/first-boot/80-lan

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+if [ -f /var/lib/liquid/lan/disable_first_boot_server ]; then
+  echo "Skipping lan server (hotspot and eth0 dhcp)"
+  exit 0
+fi
+
+(
+  echo "Configuring lan"
+  set -x
+  /opt/liquid-core/libexec/manage putconfig lan '{"dhcp-range": "10.103.0.100,10.103.0.200,72h", "eth": true, "hotspot": {"ssid": "", "password": ""}, "ip": "10.103.0.1", "netmask": "255.255.255.0"}'
```

Comment:
Could this script actually set up the LAN instead of setting the configuration for `liquid-core` to do so?

Reply:
nope. merry christmas!